js/components/agency_preview.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * AgencyPreview
+ *
+ * Displays information for a decentralized agency.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import { Agency } from '../models';
+
+function AgencyPreview({ agency, agencyComponentsForAgency, onAgencySelect }) {
+  const description = agency.mission();
+  return (
+    <div className="agency-preview usa-grid-full">
+      <div className="usa-width-one-whole">
+        <h3>{agency.name}</h3>
+      </div>
+      <div className="usa-width-one-half">
+        <div className="heading-2">
+          Make your FOIA request directly to the most relevant component:
+        </div>
+        <ul className="agency-preview_components">
+          {
+            agencyComponentsForAgency.toArray()
+              .sort((a, b) => a.title.localeCompare(b.title))
+              .map((agencyComponent) => {
+                const onSelect = () => onAgencySelect(agencyComponent);
+                return (
+                  <li key={agencyComponent.id}>
+                    <p>
```

Comment:
do we need both the `p` and the `span`?

Reply:
I had include the `p` because this was styled just like the usual body text.. however now it's not really, so I'll remove it.